dubbo-test/dubbo-test-benchmark/src/main/java/com/alibaba/dubbo/rpc/benchmark/AbstractBenchmarkClient.java

```diff
@@ -1,11 +1,5 @@
 package com.alibaba.dubbo.rpc.benchmark;
 
-/**
- * nfs-rpc
- * Apache License
- * <p>
- * http://code.google.com/p/nfs-rpc (c) 2011
- */
 
```

Comment:
pls. add apache license at the beginning

Reply:
@beiwei30  use our new apache license comments or original comments?